### ISSUE TITLE
Replace all references to Secrets with Environment variables

### ISSUE THREAD

**Issue by tmcw** [val-town/val-town-docs#27]

**Comment by Xkonti** [comment]
This is simply to rename all instances where this concept is referred as "Secrets" to refer to it as "Environment variables" instead? If that's the case I can do that :)

**Comment by stevekrouse** [comment]
Yep, that's the ticket! We have a matching ticket in the Val Town repo that should probably come first, so we know what the path we're using instead of `/secrets` is. Maybe `/environment-variables` or `/environment`.

**Comment by Xkonti** [comment]
Let me know when it's known and I'll update the docs :)

**Comment by stevekrouse** [comment]
We shipped it! https://www.val.town/settings/environment-variables
